<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_settings</class>
 <widget class="QWidget" name="Form_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_start"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_finish"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_save_change">
     <property name="text">
      <string>Сохранить изменения</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
